nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/FlowController.java

```diff
@@ -1059,6 +1060,42 @@ public ProcessorNode createProcessor(final String type, String id, final boolean
         final LogRepository logRepository = LogRepositoryFactory.getRepository(id);
         logRepository.addObserver(StandardProcessorNode.BULLETIN_OBSERVER_ID, LogLevel.WARN, new ProcessorLogObserver(getBulletinRepository(), procNode));
 
+        try {
+
+            final Class<?> procClass = processor.getClass();
+            if(procClass.isAnnotationPresent(DefaultSettings.class))
+            {
```

Comment:
There are checkstyle violations to fix:
````
[WARNING] src/main/java/org/apache/nifi/annotation/configuration/DefaultSchedule.java[5] (imports) AvoidStarImport: Using the '.*' form of import should be avoided - java.lang.annotation.*.
[WARNING] src/main/java/org/apache/nifi/annotation/configuration/DefaultSettings.java[5] (imports) AvoidStarImport: Using the '.*' form of import should be avoided - java.lang.annotation.*.
[WARNING] src/main/java/org/apache/nifi/controller/FlowController.java[1067:13] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/FlowController.java[1071:17] (blocks) RightCurly: '}' should be on the same line.
[WARNING] src/main/java/org/apache/nifi/controller/FlowController.java[1073:17] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/FlowController.java[1079:17] (blocks) RightCurly: '}' should be on the same line.
[WARNING] src/main/java/org/apache/nifi/controller/FlowController.java[1081:17] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/FlowController.java[1086:17] (blocks) RightCurly: '}' should be on the same line.
[WARNING] src/main/java/org/apache/nifi/controller/FlowController.java[1088:17] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/FlowController.java[1093:9] (blocks) RightCurly: '}' should be on the same line.
[WARNING] src/main/java/org/apache/nifi/controller/FlowController.java[1095:9] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[191:9] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[194:13] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[197:17] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[199:17] (blocks) RightCurly: '}' should be on the same line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[201:17] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[205:17] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[207:17] (blocks) RightCurly: '}' should be on the same line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[209:17] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[214:17] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[216:21] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[218:21] (blocks) RightCurly: '}' should be on the same line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[220:21] (blocks) LeftCurly: '{' should be on the previous line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[226:9] (blocks) RightCurly: '}' should be on the same line.
[WARNING] src/main/java/org/apache/nifi/controller/StandardProcessorNode.java[228:9] (blocks) LeftCurly: '{' should be on the previous line.
````

You can check it by yourself by running:
````
mvn clean install -Pcontrib-check
````